RELEASE.md

```diff
@@ -154,80 +150,65 @@ To release a new version of Spyder you need to follow these steps:
 
-* twine check dist/*
+* `twine check dist/*`
 
-* twine upload dist/*
+* `twine upload dist/*`
 
 * Check in PyPI that the new release was published correctly
 
-* git tag -a vX.X.X -m 'Release X.X.X'
+* `git tag -a vX.X.X -m "Release X.X.X"`
 
 * Update version in `__init__.py` (add 'dev0' and increment minor)
 
 * `git add .` and `git commit -m "Back to work [ci skip]"`
 
-* git checkout master
-
-* git merge 5.x
-
-* git commit -m "Release X.X.X [ci skip]"
+* `git push upstream master`
 
-* git push upstream master
-
-* git push upstream 5.x
-
-* git push upstream --tags
+* `git push upstream --tags`
 
 
 ## After the release
 
-* Publish release in our Github Releases page:
+* Publish release in our [Github Releases page](https://github.com/spyder-ide/spyder/releases):
   - Copy the contents of the previous release description (updating the relevant information and links to point to the new Spyder version and changelog entry).
   - Edit the previous release description to only have the changelog line.
 
-* Merge PR on Conda-forge for Spyder. For that you can go to
-
-  https://github.com/conda-forge/spyder-feedstock
-
-  and merge the corresponding PR for the new release (usually an automatic PR will appear that can be either merged as it is or be use as boilerplate).
+* Merge PR on Conda-forge for Spyder. For that you can go to the [spyder-feedstock repo](https://github.com/conda-forge/spyder-feedstock) and merge the corresponding PR for the new release (usually an automatic PR will appear that can be either merged as it is or be use as boilerplate).
 
   **Notes**:
 
-  - Don't forget to add new dependencies and update constraints on the rest of them. For that, you need to compare line by line the contents of the `recipe/meta.yaml` file in the feedstock with
-
-    https://github.com/spyder-ide/spyder/blob/5.x/setup.py
-  - After merging, give a ping to `@anaconda-pkg-build` about the new release.
+  - Don't forget to add new dependencies and update constraints on the rest of them. For that, you need to compare line by line the contents of the `recipe/meta.yaml` file in the feedstock with [setup.py](https://github.com/spyder-ide/spyder/blob/master/setup.py)
 
 * Update Binder related elements when the new Spyder version is available in Conda-forge:
   - Update the Spyder version on the environment file ([`binder/environment.yml`](https://github.com/spyder-ide/binder-environments/blob/spyder-stable/binder/environment.yml)) of the ([`spyder-stable` branch](https://github.com/spyder-ide/binder-environments/tree/spyder-stable)) in the `binder-environments` repo.
-  - Update `environment.yml` files of the [`master`](https://github.com/spyder-ide/binder-environments/blob/master/binder/environment.yml) and [`5.x`](https://github.com/spyder-ide/binder-environments/blob/5.x/binder/environment.yml) branches of `binder-environments` with the contents of the `binder/environment.yml` file present on this repo.
```

Comment:
Leaving a comment here to not forget to restore this part

    [`6.x`](https://github.com/spyder-ide/binder-environments/blob/6.x/binder/environment.yml)`

once 6.0 is released.